src/io/flutter/run/daemon/DaemonEvent.java

```diff
@@ -21,26 +21,25 @@
  * >The Flutter Daemon Mode</a>.
  */
 abstract class DaemonEvent {
-
   /**
    * Parses an event and sends it to the listener.
    */
   static void dispatch(@NotNull JsonObject obj, @NotNull Listener listener) {
     final JsonPrimitive primEvent = obj.getAsJsonPrimitive("event");
     if (primEvent == null) {
-      LOG.error("Missing event field in JSON from flutter process: " + obj);
+      LOG.info("Missing event field in JSON from flutter process: " + obj);
```

Comment:
why are these going from error to info log messages?

Reply:
Logging an error here will actually display a prompt to the user (a crash occurred...). This is just bad data from the wire, not a programmer error (so on the order of not being able to read a file, not an NPE), so I won't want to show a dialog.